#region Left motors
GPIO13_PWM1 = 33
GPIO19_PWM1 = 35
GPIO2 = 3
GPIO3 = 5
GPIO4 = 7
GPIO14 = 8
GPIO15 = 10
#endregion

#region Right motors
GPIO12_PWM0 = 33
GPIO18_PWM0 = 35
GPIO17 = 11
GPIO27 = 13
GPIO22 = 15
GPIO23 = 16
GPIO24 = 18
# endregion

#region IR
GPIO10 = 19
GPIO21 = 21
GPIO11 = 23
#endregion

#region Ultrasonic
GPIO5 = 29
GPIO6 = 31
#endregion

#region Misc
MAX_SPEED_MOTOR = 100
PWM_FREQUENCY = 50 # TODO: is 50 enough?
DIR_FORWARD = 1
DIR_BACKWARD = -1

ULTRASONIC_PING_PULSE_LENGTH = 10 * 10**-6 # 10us
ULTRASONIC_TIMEOUT_LENGTH = 0.05 # seconds
ULTRASONIC_MIN_RANGE = 3
ULTRASONIC_MAX_RANGE = 400
ULTRASONIC_FOV = 0.26179938779915 # 15 degrees
#endregion